src/Views/Common.hs

```diff
@@ -91,3 +93,44 @@ addChildrenToParent parent children = do
 
 addChildrenToParent' :: Elem -> [Elem] -> Client ()
 addChildrenToParent' parent children = mapM_ (appendChild parent) children
+
+-- |Creates an input field and button. Has 'String' as id and '(Client ())' is the function
+-- |That activates when clicking button or pressing enter. Descriptive text is second 'String'
+-- |Div is created with id = identifier ++ "Div". Field is created with id = identifier ++ "Field".
+-- |Button is created with id = identifier ++ "Btn"
+-- |The field and button is then placed in the right sidebar.
+createInputFieldWithButton :: String -> String -> Client () -> Client ()
+createInputFieldWithButton identifier text function = do
+  parentDiv <- createDiv [("id", identifier ++ "Div")]
```

Comment:
It's common practise to use lower-case together with hyphens when declaring html id:s.
So `identifier ++ "Div"` becomes `identifier ++ "-div"`.